lib/resty/upstream/healthcheck.lua

```diff
@@ -597,6 +605,16 @@ function _M.spawn_checker(opts)
         return nil, "no upstream specified"
     end
 
+    local hook_down = opts.hook_down
```

Comment:
Just rolling back to this, I haven't changed this largely to keep in line with existing style. I'd be on board with a separate PR to update this if necessary. Just wanted to touch base on why this hasn't been touched.